api-client/src/main/java/com/takipi/api/client/request/team/GetTeamMembersRequest.java

```diff
@@ -0,0 +1,45 @@
+package com.takipi.api.client.request.team;
+
+import com.takipi.api.client.data.team.ServiceUsers;
+import com.takipi.api.client.request.ServiceRequest;
+import com.takipi.api.core.request.intf.ApiGetRequest;
+
+public class GetTeamMembersRequest extends ServiceRequest implements ApiGetRequest<ServiceUsers>

```

Comment:
Why isn't this `ServiceUsersResponse`?

Reply:
different field names.

Reply:
In that case I'd rename `ServiceUsers` and `ServiceUsersResponse` to `TeamMembersResult` and `NewTeamMembersResult` so it's more clear which is which.